libs/zephyr-rspress-plugin/src/with-zephyr.ts

```diff
@@ -0,0 +1,19 @@
+import type { RspressPlugin } from '@rspress/shared';
+import { zephyrRspressSSGPlugin } from './zephyrRspressSSGPlugin';
+import { zephyrRsbuildPlugin } from './zephyrRsbuildPlugin';
+
+export function withZephyr(): RspressPlugin {
+  return {
+    name: 'plugin-zephyr-rspress',
```

Comment:
can we follow our naming standard?
```suggestion
    name: 'zephyr-rspress-plugin',
```